pkgs/servers/ftp/vsftpd/default.nix

```diff
@@ -24,7 +24,11 @@ stdenv.mkDerivation rec {
   '';
 
   NIX_LDFLAGS = "-lcrypt -lssl -lcrypto -lpam -lcap";
-  NIX_CFLAGS_COMPILE = "-Wno-error=enum-conversion";
+
+  # On gcc9, this would produce
+  #   error: '-Werror=enum-conversion': no option -Wenum-conversion
+  NIX_CFLAGS_COMPILE = lib.optionalString (lib.versionAtLeast stdenv.cc.version "10")
```

Comment:
In cases like this it might be nicer to _also_ test that it's gcc for the odd person who comes along and tries compiling with clang...

Reply:
I tested compiling with clang and then it complains about gcc not being found so I did not bother with that.

Reply:
Fair enough I guess.